### ISSUE TITLE
IDE0044 (add readonly modifier) fires incorrectly for tuple assignment

### ISSUE THREAD

**Issue by bkoelman** [dotnet/roslyn#27836]
**Version Used**: 
VS 2017 Community v15.7.3

**Steps to Reproduce**:
The next program reports IDE0044 with an empty description:
```csharp
class C
{
    private int field1; // IDE0044
    private int field2; // IDE0044

    void M()
    {
        (field1, field2) = GetTuple();

        Console.WriteLine($"{field1}, {field2}");
    }

    private (int, int) GetTuple() => throw null;
}
```

**Expected Behavior**:
No report of IDE0044, because an assignment is made outside of a constructor or initializer.

**Actual Behavior**:
IDE0044 reported on `field1` and `field2`. Lightbulk-menu has option "Add readonly modifier", which results in compile error CS0191: A readonly field cannot be assigned to.

**Comment by Neme12** [comment]
duplicate of #26264

**Comment by bkoelman** [comment]
@Neme12 Thanks, I will close this one then.

**Comment by Neme12** [comment]
This is fixed in 15.8 Preview 2
